googlemock/test/gmock-matchers_test.cc

```diff
@@ -2778,7 +2778,7 @@ class FloatingPointTest : public testing::Test {
 
   // Pre-calculated numbers to be used by the tests.
 
-  const size_t max_ulps_;
+  const unsigned int max_ulps_;
```

Comment:
I showed the warnings here: https://github.com/mark-chromium/googletest/commit/8b085f0d2168e6e54712d295dcf26feac92bb9fc

They’re all of the form

        close_to_positive_zero_(
            Floating::ReinterpretBits(zero_bits_ + max_ulps_/2)),

With RawType = float, Bits (the type of zero_bits_ and ReinterpretBits()’ argument type) is unsigned int. Because max_ulps_ is size_t, with a 64-bit size_t, the expression for the argument is computed as size_t, and then the compiler issues a warning about narrowing to unsigned int to pass to ReinterpretBits().